sync/sync.go

```diff
@@ -83,3 +102,40 @@ func (s *String) Set(value string) {
 	defer s.rw.Unlock()
 	s.value = value
 }
+
+// Print returns string representation of value.
+func (s *String) Print() string {
```

Comment:
If we use the std Stringer interface of Go we can actually make the logging part even easier because it will be invoked automatically:
```log.Infof("seed value %v applied on field %s", f, f.Name())```

https://golang.org/pkg/fmt/

```
3. If the %v verb is used with the # flag (%#v) and the operand implements the GoStringer interface, that will be invoked.
```